pkg/controller/cloud/node_controller.go

```diff
@@ -148,6 +148,14 @@ func (cnc *CloudNodeController) updateNodeAddress(node *v1.Node, instances cloud
 		return
 	}
 
+	//Node not present according the cloud provider, no need to update node address.
```

Comment:
nit: `// Node that isn't present according to the cloud provider shouldn't have its address updated`

Reply:
Thanks your review,  I will fix it.